app/views/layouts/application.html.haml

```diff
@@ -109,5 +108,4 @@
                 %a.govuk-footer__link{href: "https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/", rel: "license"} Open Government Licence v3.0
           .govuk-footer__meta-item
             %a.govuk-footer__link.govuk-footer__copyright-logo{href: "https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/"} © Crown copyright
-  :javascript
-    window.GOVUKFrontend.initAll()
+    = javascript_pack_tag 'application'
```

Comment:
I don't know how this will behave at build time given that it's an integration between webpack and rails. If you refactor `application.js` to use code splitting and this was a modern Single Page App, I'd expect webpack to inject a collection of separate `<script src` and `<link preload` tags, one for each `import(...` declaration. Webpack would use the comments to add the name to the file, and would also automatically include a cachebusting hash.